### PR TITLE
Add TravisCI support

### PR DESCRIPTION
**Pull request by z4y4ts**

### CODE REVIEW

File: requirements_test.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+webtest==2.0.15
+simplejson==3.5.3
+numpy==1.8.1
+scipy==0.14.0
+colander==1.0b1
+sphinx==1.2.2
+breathe==1.2.0
+sphinxcontrib-httpdomain==1.2.1
+sphinx_rtd_theme==0.1.6
+cma
```
Comment (z4y4ts) [inline requirements_test.txt]: @idegtiarov can you please point what version do we need  here?

Comment (z4y4ts) [inline requirements_test.txt]: Also we need to review `requirements.txt` to wipe out any redundant dependencies and pin version.

Comment (z4y4ts) [inline requirements_test.txt]: @idegtiarov will you update `requirements.txt` in this PR or in new one?

**Comment by z4y4ts** [comment]
@idegtiarov will you update requirements.txt in this PR or in new one?

**Comment by idegtiarov** [comment]
Good catch. Remove all odd packages from the requirements

File: edx_adapt/settings.py [hunk 2]
```diff
@@ -1 +1 @@
-LOGS_DIR = '/var/log/edx-adapt/'
+LOGS_DIR = 'log/edx-adapt/'
```
Comment (z4y4ts) [inline edx_adapt/settings.py]: @idegtiarov Can you please confirm this is ok?

File: requirements.txt [hunk 3]
```diff
@@ -1,8 +1,5 @@
-boto
 flask
 flask-cors
 flask_restful
-hmmlearn
 pymongo
-sklearn
 requests
```
Comment (z4y4ts) [inline requirements.txt]: Please pin versions. I.e. the ones we currently use.

File: edx_adapt/settings.py (hunk 2 above)
Comment (idegtiarov) [inline edx_adapt/settings.py]: We should change permission for /var/log/edx-adapt dir in travis config or write logs into project dir while runing travis tests

**Pull request merged**